### ISSUE TITLE
Pre-head complements in nominals

### ISSUE THREAD

**Issue by nschneid** [nert-nlp/cgel#67]
I was surprised to find that CGEL (starting on p. 439) regards some pre-head dependents in nominals as complements rather than attributive modifiers. I haven't seen computational linguists make this distinction as part of syntax and I assume we're not making it in CGELBank.

**Comment by BrettRey** [comment]
Yes, I have a feeling that John has backed off on that, but I'm not sure
about it. I haven't been making the distinction. Do you think we should? I
suppose if we don't we should mention it in the guidelines.

**Comment by nschneid** [comment]
Yeah I think we should document it as something we're not attempting. SIEG2 p. 128: "Complements in NP structure are therefore virtually restricted to PPs and subordinate clauses in the head nominal."

**Comment by BrettRey** [comment]
I'll do that.

**Comment by BrettRey** [comment]
2.4.2 Modifier vs Complement
CGEL (starting on p. 439) regards some pre-head dependents in nominals as complements rather than attributive modifiers, as in a flower seller or an income tax adviser. In contrast, we analyze these pre-head dependents as modifiers.
